sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/SearchableFieldBuilder.java

```diff
@@ -31,19 +31,19 @@ public SearchableField(String name, boolean collection) {
      *
      * @return The {@link LexicalAnalyzerName} used for analyzer.
      */
-    public LexicalAnalyzerName getAnalyzer() {
-        return analyzer;
+    public LexicalAnalyzerName getAnalyzerName() {
+        return analyzerName;
     }
 
     /**
      * Sets the name of the language analyzer. This property cannot be set when either {@code searchAnalyzer} or
      * {@code indexAnalyzer} are set. Once the analyzer is chosen, it cannot be changed for the field in the index.
      *
-     * @param analyzer The {@link LexicalAnalyzerName} used for analyzer.
+     * @param analyzerName The {@link LexicalAnalyzerName} used for analyzer.
      * @return The SearchableField object itself.
```

Comment:
Update javadoc return type in all methods of this class.
```suggestion
     * @return The SearchableFieldBuilder object itself.
```